### DISCUSSION TITLE
Welcome to GitHub Discussions

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkusio/quarkus#21879] [Announcements]
Here you can discuss (almost) any topic around Quarkus community!

We still use quarkus-dev for Quarkus core development discussions and but everything else we ask to do here.

## Categories

We've setup some initial categories to get things started.

Two general categories:

**Community** is the "if it does not fit anywhere else post here. 
**Q&A** is for Quarkus usage questions. Similar to what you can use Stackoverflow for.

Then we have 3 categories for content we've seen post in various places but we find useful 
to keep separate so easier to find and more focused: 
**Introductions** - for those who are new (or old) and would like to  introduce them selves
**Events** - is for discussing conferences, trainings, podcasts, books related to Quarkus
**Jobs and Opportunities** - if you are looking for people to work on Quarkus related projects

Finally two special ones:
**Annoucements** we use for making announcements like this one.
**Quarkus Blog/Website** is used for comments from https://quarkus.io.

## Labels

If you are a contributor with triage permissions you can use labels to classify your discussions - i.e. "area/cli" if your question is about Quarkus CLI or "area/camel" if its about Camel. If you find there is not a relevant label let us know and we can add it.

We do plan on enabling @quarkusbot to help classifying the posts so don't be surprised if it adds labels automatically for you.

Thanks!

**Comment by edeandrea** [comment]
FYI I don't think "regular" users are able to add labels on their own. I am not able to add labels to a discussion.

**Reply by maxandersen** [reply]
You should be able to during creation ?

**Reply by edeandrea** [reply]
I don't see a way to
![Screen Shot 2021-12-08 at 8 45 06 AM](https://user-images.githubusercontent.com/363447/145218814-32f2d39c-8bdc-45a6-9b1a-01a490c854ae.png)

And in a discussion I already created the label area on the right doesn't allow me to add labels.

![Screen Shot 2021-12-08 at 8 46 18 AM](https://user-images.githubusercontent.com/363447/145218977-8ee4df82-cd62-4266-a812-125dda872e79.png)

**Reply by maxandersen** [reply]
you are right - that is an annoying limitation I hope github will fix eventually. Updated the text to reflect reality!

**Comment by ghost** [comment]
If i want to learn quarkus, i have to learn Vertex first ?

**Reply by maxandersen** [reply]
Hi,

No, you do not need to learn vert.x to get started with Quarkus. 

Just go to https://quarkus.io/get-started/ and follow the instructions and links to deeper guide as you progress.

**Comment by ghost** [comment]
Hi , I am facing - 

java.io.IOException: Cannot run program "C:\Program Files\Java\jdk-17.0.2\bin" (in directory "D:\quarkus\devtools\project-core-extension-codestarts\target\classes\gradle-wrapper"): CreateProcess error=5, Access is denied

and 

Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:exec (generate-gradle-wrapper) on project quarkus-project-core-extension-codestarts: Command execution failed.: Cannot run program "C:\Program Files\Java\jdk-17.0.2\bin" (in directory "D:\quarkus\devtools\project-core-extension-codestarts\target\classes\gradle-wrapper"): CreateProcess error=5, Access is denied -> [Help 1]

I am facing this exception can you help me to resolve this?
